admin_guide/managing_networking.adoc

```diff
@@ -674,7 +678,13 @@ ifdef::openshift-origin[]
     image: openshift/origin-pod
 endif::openshift-origin[]
 ----
-<1> This uses the YAML syntax for a multi-line string; see below for
+<1> IP address from the physical network that the node is on and is reserved by the
+cluster administrator for use by this pod. Optionally, you can include the
+subnet length (the `/24`) so that a proper route to the local subnet can be set
+up. If you do not specify a subnet length, then the egress router can access
+only the host specified with the `EGRESS_GATEWAY` variable, and no other hosts
+on the subnet.
+<2> This uses the YAML syntax for a multi-line string; see below for
```

Comment:
I might move the annotation up to the same line as EGRESS_DESTINATION and say something like "\<2> Provide `EGRESS_DESTINATION` values as multi-line YAML string. Use the following formats:
ul goes here"